lib/utils.ts

```diff
@@ -0,0 +1,20 @@
+export const keysToCamel = (o: any): any => {
```

Comment:
Seems like Axios should have a way to do snake to camel?

Reply:
No idea, can look into it :+1:

Reply:
It does, i recall using a library for this but cant remember the name now

Reply:
I didn't find it within Axios but I found another one called `humps` that does this. Just replaced it.